persia/ctx.py

```diff
@@ -375,3 +400,40 @@ def on_after_backward(
                 f"current batch grad empty num: {len(empty_grad)}, {empty_grad}"
             )
         return finite
+
+    def dump_embedding(self, dst_dir: str, blocking: bool = False):
+        self.backend.dump_embedding(dst_dir, blocking)
+
+    def load_embedding(self, dst_dir: str, blocking: bool = True):
+        self.backend.load_embedding(dst_dir, blocking)
+
+
+class InferCtx(BaseCtx):
+    r"""Inference context that provide full feature of embedding lookup
+
+    Arguments:
+        backend_worker_size (int): rpc client thread pool size
+        middleware_addr([str]): addr of middlewares
+
+    """
+
+    def __init__(
+        self,
+        backend_worker_size: int = 20,
+        middleware_addrs=get_middleware_services(),
```

Comment:
Do not use funciton return val as default_value due to the function is invoke at module import time but not the `InferCtx ` runtime

Reply:
Solved in https://github.com/PersiaML/PersiaML/pull/18/commits/44c21ffae2b6a3895503cc6bfd256a2d3d8645f0